jerry-ext/debugger/debugger-tcp.c

```diff
@@ -144,6 +144,26 @@ jerryx_debugger_tcp_close (jerry_debugger_transport_header_t *header_p) /**< tcp
 
   jerryx_debugger_transport_tcp_t *tcp_p = (jerryx_debugger_transport_tcp_t *) header_p;
 
+  /* Waiting the debug client close the tcp connection first */
+  while (true)
+  {
+    char buf[8];
+    jerryx_socket_ssize_t result = recv (tcp_p->tcp_socket, buf, sizeof (buf), 0);
+    if (result == 0)
+    {
+      /**
+       * If result >= 0, this means that there is either data available on the socket, or the socket
+       * has been closed. So waiting the socket closed by remote client by result == 0
```

Comment:
This part breaks the loop. Where is the 'waiting' part?

Reply:
== 0 means remote closed the tcp socket, > 1 means still have data, < 0 means have error, JERRYX_EWOULDBLOCK error should be ignored(means have no data but the tcp socket still there)